elixir/apps/web/lib/web/live/relay_groups/edit.ex

```diff
@@ -3,11 +3,13 @@ defmodule Web.RelayGroups.Edit do
   alias Domain.Relays
 
   def mount(%{"id" => id}, _session, socket) do
-    with {:ok, group} <- Relays.fetch_group_by_id(id, socket.assigns.subject) do
+    with true <- Domain.Config.relay_admin_enabled?(),
+         {:ok, group} <- Relays.fetch_group_by_id(id, socket.assigns.subject) do
       changeset = Relays.change_group(group)
       {:ok, assign(socket, group: group, form: to_form(changeset))}
     else
       {:error, _reason} -> raise Web.LiveErrors.NotFoundError
+      _feature_disabled -> raise Web.LiveErrors.NotFoundError
```

Comment:
No need to be repetitive here:
```suggestion
      _other -> raise Web.LiveErrors.NotFoundError
```

Reply:
Yeah, I had considered that.  The reason I chose to duplicate was because I knew the `_feature_disabled` would be going away (hopefully in short order).  With that in mind are you ok with using `_other` after the feature flag is gone and the feature is in permanently?  Happy to make this change, just wanted to confirm.

Reply:
I went ahead and just changed it to your suggestion.  If my previous comment is an issue we can always change it back when we remove the feature flag in question.